### ISSUE TITLE
Postgres SiteTree_Localised  column "locale" does not exist

### ISSUE THREAD

**Issue by wojciechmruk** [tractorcow-farm/silverstripe-fluent#505]
Hi, 
I'm using Postgres 9.2, Silverstripe 4.3 and fluent version is 4.2.0

I've added a fluent module and dev/bulid?flush=all

Here is an error:

`[Emergency] Uncaught SilverStripe\ORM\Connect\DatabaseException: Couldn't run query: SELECT "RecordID" FROM "SiteTree_Localised" WHERE (Locale = $1) ERROR: column "locale" does not exist LINE 3: WHERE (Locale = $1) ^
`

I Believe that the problem is here: \tractorcow\silverstripe-fluent\src\Extension\FluentVersionedExtension.php in "prepoulateIdsInLocale" method.

Line: 424
Original:

```
            $select = SQLSelect::create(
                ['"RecordID"'],
                '"' . $table . '"',
                ['Locale' => $locale]
            );
```

It is fine for MySql but not for Postgres and I think it should look like this:

```
            $select = SQLSelect::create(
                ['"RecordID"'],
                '"' . $table . '"',
                ['"Locale"' => $locale]
            );
```

Locale key should have "" like RecordID and $table has.

I've tested only on Postgres and it works for me.

**Comment by robbieaverill** [comment]
Thanks for the bug report. Would you like to change this into a pull request? We have unit tests that run in Postgres too, so I assume that this part of the code isn't covered by existing unit tests. If you could add a new unit test to cover this code too that'd be even better, but a patch would be great anyway!

**Comment by tractorcow** [comment]
Yes absolutely good catch. All SQL identifiers must be double quoted. It's only optional in MYQSL but mandatory in PostgreSQL.

**Comment by robbieaverill** [comment]
Fixed in #506
